### ISSUE TITLE
DirectX 11 changes keyboard layout

### ISSUE THREAD

**Issue by Duion** [GarageGames/Torque3D#1752]
When I run the game in DirectX 11 mode, then my keyboard layout is changed to US.
For those who do not know how a german keyboard works, it basically switches y and z.
So when I run the game in DirectX 9 z is z and y is y for me, but with DirectX 11 suddenly z is y and y is z, can be quite confusing.
Any ideas why this could be the case?

**Comment by Bloodknight** [comment]
maybe the console key is working correctly again now too :p 
There was a couple of past issues, one of which involved people who had different language systems installed. Another was a pure arrogance issue from another company forcing everyone to use the US keyboard system.

I seem to recall there was a way to change it in a PR early in the life of T3D MIT in torque.rc.

**Comment by Duion** [comment]
Yes the console key works now as well, but I think the DX11 layout is the wrong one.

**Comment by Duion** [comment]
It does seem that my keyboard layout is now US always, not just in DX11.
